### DISCUSSION TITLE
使用webdave，下面这些场景会如何处理？

### DISCUSSION THREAD

**Discussion by kof8855** [Jeric-X/SyncClipboard#119] [Q&A]
场景1：电脑A和电脑B同时使用，并且都连接到相同webdav服务器，A和B同时复制、各自复制，会不会有冲突？
场景2：电脑在外没有联网连接webdav，离线状态办公操作，当有网络之后，会不会同步更新无网络期间的剪切记录到webdav？

**Comment by Jeric-X** [comment]
首先，服务器永远只会存储最新的一条记录，即任意客户端最后一次上传的那条记录
1. 这取决于服务端如何实现原子性，就结果来说只会保留一条记录，其中一台电脑会被覆盖，复制的内容丢失
2. 不会，在联网后触发新的复制后会向服务器更新最新的内容
